API-DOCS/USER.md

```diff
@@ -42,9 +35,7 @@
 #### User logs in to their account:
 
 ```js
-axios.post('/person/register', {
-    token: '<Google OAuth Token>'
-}).then(function (response) {
+axios.post('/person/register', {}).then(function (response) {
```

Comment:
I think the endpoint should be /register, also could you specify that this relies solely on the Auth Token present in the headers section

Reply:
/user/register? And yes, I can add a note about that for any posts that don't pass an id.